packages/@webex/plugin-meetings/src/locus-info/selfUtils.ts

```diff
@@ -159,6 +162,9 @@ SelfUtils.breakoutsChanged = (previous, current) =>
 SelfUtils.interpretationChanged = (previous, current) =>
   !isEqual(previous?.interpretation, current?.interpretation) && !!current?.interpretation;
 
+SelfUtils.brbChanged = (previous, current) =>
+  !isEqual(previous?.brb, current?.brb) && !!current?.brb;
```

Comment:
```suggestion
SelfUtils.brbChanged = (previous, current) =>
  !isEqual(previous?.brb, current?.brb) && current?.brb !== undefined;
```

Because with current code:
```
(previous -> current | result)
undefined -> true    | brbChanged = true
true      -> false   | brbChanged = false // this is not true
false     -> true    | brbChanged = true
```

Catched with unit tests

Reply:
Great catch, will update this part